demo/src/app/components/popover/popover.component.ts

```diff
@@ -0,0 +1,11 @@
+import {Component} from '@angular/core';
+import {NgbdApiDocs} from '../shared/api-docs';
+
+@Component({
+  selector: 'ngbd-popover',
+  template: `
+    <ngbd-api-docs directive="NgbPopoverWindow"></ngbd-api-docs>
```

Comment:
We don't want to document `NgbPopoverWindow` as those are internal components not exposed to users. Remove.